calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -19,11 +21,21 @@
 
 from netaddr import IPAddress, IPNetwork
 
+from pycalico.util import generate_cali_interface_name
+
+
 VETH_NAME = "eth1"
-"""The name to give to the veth in the target container's namespace. Default
-to eth1 because eth0 could be in use"""
+"""
+The name to give to the veth in the target container's namespace. Default
+to eth1 because eth0 could be in use.
```

Comment:
This name is only used when netns.py is used to move the veth into the namespace and _not_ when libnetwork is used.